R/AbstractPackageReporter.R

```diff
@@ -81,55 +81,37 @@ AbstractPackageReporter <- R6::R6Class(
             
             return(invisible(NULL))
         },
+        get_summary_view  = function(){
+            stop("get_summary_view has not been implemented.")
+        }
+    ),
+    
+    active = list(
         
-        get_package = function(){
+        package_name = function(){
             return(private$packageName)
         },
-   
-        get_report =  function(output_file = NULL) {
-            rmarkdown::render(self$get_report_markdown_path()
-                              , output_format = "html_document"
-                              , output_file = output_file
-                              , quiet = TRUE
-                              , envir = new.env()
-                              , params = list(reporter = self))
-        },
-        
-
-        get_package_path = function(){
-            return(private$packagePath)
-        },
         
-        get_report_markdown_path =  function() {
-            stop("get_report_markdown_path has not been implemented.")
-        },
-        
-        get_summary_view  = function(){
-            stop("get_summary_view has not been implemented.")
-        },
-
-        get_raw_data = function(){
```

Comment:
Ohhh,  I didn't realize this was a thing we created.  yeah, good call on removing it.